docs/source/intro/installation.rst

```diff
@@ -40,6 +40,11 @@ To assume the role of ``postgres`` run as root:
 
    $ psql
 
+.. tip::
+
+   If you see an error message like ``psql: FATAL:  role "<role_name>" does not exist``, probably you don't have any role created yet.
+   You can use ``-d template1`` to connect to the default database and create a new role.
+
```

Comment:
I use postgresql from conda, can anyone who use system postgresql test that `psql -d template1` will always work, then this tip is not need but can directly put `psql -d template1` as the command.

Reply:
I can confirm that `psql -d template1` allows to connect as the `postgres` user when installed as a system wide service. However, there _are_ some roles present. I don't want to delete them all to check whether this would still work if no roles exist. But I think we can keep this and assume it works.

Reply:
I don't really understand how this would fix the problem though. The error you mention I get for example when I try to run `psql` as my own user:
```
sph@invader:$ psql -d template1
psql: error: FATAL:  role "sph" does not exist
```
but that is because `psql` will use the linux system user if not explicitly specified, and my user does not have a correpsonding role to access. So how would switching the database to connect to fix that? As you can see, it doesn't for my example.